charts/cromiam/values.yaml

```diff
@@ -48,7 +48,17 @@ probes:
       periodSeconds: 10
       failureThreshold: 30 # 5 minutes before restarted
       successThreshold: 1
-
+  startup:
```

Comment:
Ah, this is broken. It looks like [this change](https://github.com/broadinstitute/terra-helm/pull/297/files) from a few months ago broke cromiam's probe config, but the change was actually never deployed.

Let's correct `/version` to `/engine/v1/version` and `/status` to `/engine/v1/status` for all the cromiam probes.

Reply:
a75d7d9